pennylane_lightning_gpu/src/simulator/cuGates_host.hpp

```diff
@@ -628,6 +626,699 @@ static auto getControlledPhaseShift(const std::vector<U> &params)
     return getControlledPhaseShift<CFP_t>(params.front());
 }
 
-} // namespace cuGates
-} // namespace CUDA
-} // namespace Pennylane
+/**
+ * @brief Create a matrix representation of the single excitation rotation
+ * gate data in row-major format.
+ *
+ * @tparam CFP_t Required precision of gate (`float` or `double`).
+ * @tparam U Required precision of parameter (`float` or `double`).
+ * @param angle Phase shift angle.
+ * @return const std::vector<CFP_t> Return const single excitation rotation
```

Comment:
Hi @maliasadi. Why is the return type `constant` here and in the next functions?

Reply:
Thanks @AmintorDusko! There shouldn't be any. Fixed!